### ISSUE TITLE
Question about your experimental results.

### ISSUE THREAD

**Issue by marsggbo** [mr7495/COVID-CTset#2]
First, I'd like to thank you for your great contribution. I have several questions:

1. I find that you split the dataset into five folders, I wonder why you do this? Why not directly use 80% data as the training set and 20% as the validation set?
2. In the `CSV` folder you provided, there are `train.csv`, `validation.csv`, `test.csv`, but I find that the `validation.csv` is a subset of `test.csv`, then what's the difference between `validation.csv` and `test.csv`?

I'm looking forward to your reply. Thanks!

**Comment by mr7495** [comment]
Hello dear researcher,
Thank you for your interest in our work.
1- Using Folds is a technique in machine learning that is used to report real performance accuracy. For example, if you like to split your data into 80% training and 20% testing, you divide your can dataset into five different folders (Five Folds), which each fold uses one folder for the test set, and the rest will be used for training. 
Due to the fact that some easier samples may be stored in the test set and so the accuracy shall be increased mistakenly, we use folds to say that our reported accuracy is the average value between folds and so it is real.
2- You are right, the validation files are subsets of test files. Validation files are used to evaluate the trained models after each epoch, so we know that after which epoch, the model has converged better. In large datasets, it is usually common to use 80,20,10  percent of data for the train, test, and validation sets. Because if the whole test set is used for evaluating the trained model after each epoch, the training procedure would be extremely time-consuming. So for this same reason, we used a part of the test set for evaluating the model while training (validation set). The reason we did not separate the validation set from the test set was that our data was not that large and it is unnecessary by the way because the final goal is to find out how the model classifies the new data that it has not seen while training and both the validation and testing sets are new to the model. So using the whole test set (including validation set) for reporting the model performance is better.
